livingatlas/pipelines/pom.xml

```diff
@@ -0,0 +1,890 @@
+                <relocation>
+                  <pattern>org.apache.http</pattern>
+                  <shadedPattern>hc45.org.apache.http</shadedPattern>
+                </relocation>
+                <!-- Transient from core: conflicts with Hadoop on Spark -->
+                <relocation>
+                  <pattern>org.hsqldb</pattern>
+                  <shadedPattern>h228.org.hsqldb</shadedPattern>
+                </relocation>
+                <relocation>
+                  <pattern>com.google.common</pattern>
+                  <shadedPattern>g20.com.google.common</shadedPattern>
+                </relocation>
+              </relocations>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>docker-maven-plugin</artifactId>
+        <version>${io.fabric8.version}</version>
+        <extensions>true</extensions>
+        <executions>
+          <execution>
+            <id>start</id>
+            <phase>pre-integration-test</phase>
+            <goals>
+              <goal>start</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>stop</id>
+            <phase>post-integration-test</phase>
+            <goals>
+              <goal>stop</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <images>
+            <image>
+              <alias>solr8</alias>
+              <name>djtfmartin/biocache-pipelines-solr8:v3</name>
+              <external>
+                <type>compose</type>
+                <composeFile>solr8.yml</composeFile>
+              </external>
+            </image>
+            <image>
+              <alias>ala-nameservice</alias>
+              <name>djtfmartin/ala-namematching-service:v20200214-4</name>
+              <external>
+                <type>compose</type>
+                <composeFile>ala-nameservice.yml</composeFile>
+              </external>
+              <run>
+                <wait>
+                  <http>
+                    <url>http://localhost:9179/api/search?q=Acacia</url>
+                    <method>GET</method>
+                    <status>200</status>
+                  </http>
+                  <time>20000</time>
+                </wait>
+              </run>
+            </image>
+          </images>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <properties>
+    <!-- Common variables -->
+    <jdkLevel>1.8</jdkLevel>
+    <encoding>UTF-8</encoding>
+    <surefireArgLine>-Xmx1024m</surefireArgLine>
+
+    <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
+    <project.resources.sourceEncoding>${encoding}</project.resources.sourceEncoding>
+
+    <!-- Tools-->
+    <gbif-pipelines.version>2.5.4-SNAPSHOT</gbif-pipelines.version>
```

Comment:
This seems suspicious, as the module is part of the gbif-pipelines project

Reply:
The version of the module is set to `1.0-SNAPSHOT`. If this is not set, then maven assumes the gbif dependencies are `1.0-SNAPSHOT` as well.

Reply:
> The version of the module is set to 1.0-SNAPSHOT.

I added a comment that also seems wrong. This module is part of the 2.5.4 release of pipelines, and I think it'll be confusing to vary versions across modules and might break the current release tools. Are there examples of other projects doing this?

Reply:
I thought this was being done with a module called `ingest-hdfs-table` that i thought was within this project. Perhaps its now been moved ?

Reply:
ive switched to 2.5.4-SNAPSHOT